invenio_search/ext.py

```diff
@@ -12,14 +13,10 @@
 import json
 import os
 import warnings
+from importlib.resources import files
```

Comment:
```suggestion
import importlib
```

minor: minor preference, since `files` is pretty ambiguous (and not clear if a function or variable name)